modules/EmbedPlayer/resources/mw.EmbedPlayer.js

```diff
@@ -3072,6 +3072,8 @@
 		handlePlayerError: function (data, shouldHandlePlayerError) {
 			if (this.shouldHandlePlayerError || shouldHandlePlayerError) {
 				var message = data ? data : this.getKalturaMsg('ks-CLIP_NOT_FOUND');
+				/* there are two formats used to represent error messages*/
+				message = message.errorMessage ? message.errorMessage : message;
```

Comment:
@InbarItay lets have one more check - if message not defined - use the default (clip_NOT_FOUND)...